massa-factory-worker/src/endorsement_factory.rs

```diff
@@ -187,7 +187,7 @@ impl EndorsementFactoryWorker {
                     index: index as u32,
                     endorsed_block,
                 },
-                EndorsementSerializer::new(), // TODO reuse self.endorsement_serializer
```

Comment:
I think this todo should still be here. The goal is too avoid initializing a new serializer each loop run

Reply:
I've added it again. Maybe I will adress the issue in this PR too...

Reply:
Field added to EndorsementFactoryWorker